homeassistant/components/zha/helpers.py

```diff
@@ -617,9 +617,11 @@ def handle_raw_device_initialized(self, event: RawDeviceInitializedEvent) -> Non
                     ATTR_NWK: str(event.device_info.nwk),
                     ATTR_IEEE: str(event.device_info.ieee),
                     DEVICE_PAIRING_STATUS: event.device_info.pairing_status.name,
-                    ATTR_MODEL: event.device_info.model
-                    if event.device_info.model
-                    else UNKNOWN_MODEL,
+                    ATTR_MODEL: (
```

Comment:
This looks like a cosmetic change and probably should be in another PR.

Reply:
This was done by the ruff formatter I believe because of the other change in the statement.